config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AlgaePond-Ds_ok-hRS2ODJNs3LUP9kw==.json

```diff
@@ -69,27 +69,17 @@
           "OreDict:8": ""
         },
         "1:10": {
-          "id:8": "miscutils:item.BasicAgrichemItem",
```

Comment:
why did you remove them? that makes no sense. I even explained that to you. please revert that

Reply:
Because they're unobtainable items at the first tiers of the Algae Farm, which are what the quest is for. In general, quests don't give unobtainable items as rewards with few exceptions, so I automatically removed all algae but the first one. Didn't think about it.

Reply:
i dont the algae by themselves are particularly useful to skip anything. you can make it just 1 if you think they are.